factorio

```diff
@@ -483,7 +483,7 @@ function install(){
     fi
     
     # parse the response
-    if filename=$(echo "${httpresponse}" |grep -oP '(?<=^location: )[^\?]+' |grep -oP 'factorio_headless.+'); then
+    if filename=$(echo "${httpresponse}" |grep -oP '(?<=^location: )[^\?]+' |grep -oP 'factorio-headless.+'); then
```

Comment:
`-` is a special character in PCRE (Perl regex): `'factorio-headless.+'`

The following should allow both to co-exist: `'factorio[_-]headless.+'`